### PR TITLE
Made and added Hatch_Input_ME

### PR DESCRIPTION
**Pull request by this-til**
be similar to GT_MetaTileEntity_Hatch_InputBus_ME , but it is fluid version
Cache_67679f636dd3697b

### CODE REVIEW

**Review by miozune — CHANGES_REQUESTED** [review]
This has the same issue as stocking bus: `Actionable.SIMULATE` might return more fluids than `Actionable.MODULATE` actually can provide, resulting in dupe of fluids.
The simple solution would be actually storing fluids in the hatch, but it removes "load balancer" functionality. One solution to this would be to allow setting limit to draw from network per each fluid type.
But I guess it will be a hard work for you... You can ask questions on #mod-dev channel on Discord, or wait for one of us fixing stocking bus and adapt changes from there.

**Comment by harrynull** [comment]
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12269

**Comment by Dream-Master** [comment]
@this-til can you run spotless

File: src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java [hunk 1]
```diff
@@ -1893,6 +1885,11 @@ private static void registerAE2Hatches() {
                     INPUT_BUS_ME.ID,
                     "hatch.input_bus.me",
                     "Stocking Input Bus (ME)").getStackForm(1L));
+            ItemList.Hatch_Input_ME.set(
+                new GT_MetaTileEntity_Hatch_Input_ME(
+                    INPUT_ME.ID,
+                    "hatch.input.me",
+                    "Stocking Input (ME)").getStackForm(1L));
```
Comment (Jonodonozym) [inline src/main/java/gregtech/loaders/preload/GT_Loader_MetaTileEntities.java:1892]: Stocking Input (ME) -> Stocking Input Hatch (ME)

**Comment by Dream-Master** [comment]
@this-til can you take care your pr and fix the stuff so we can re review it ?  thanks

**Comment by Pilad** [comment]
> be similar to GT_MetaTileEntity_Hatch_InputBus_ME , but it is fluid version Cache_67679f636dd3697b
Here are the fixes added. Does your code need to be written in the same way or not? Nothing will break in the operation of Hatch_Input_ME?
https://github.com/GTNewHorizons/GT5-Unofficial/commit/6599b089a7d259ddf0313b10e26e00ab116d3699
https://github.com/GTNewHorizons/GT5-Unofficial/commit/b893dfcaae9f1e2186073606fce1f7b79e3c78bb

**Comment by FoKuCNiK** [comment]
When will this candy be released?

**Comment by Dream-Master** [comment]
> When will this candy be released?

Go and fin user @this-til to finished the pr and resolve conflicts

**Comment by tth05** [comment]
@Dream-Master Considering OP seems to have gone missing, I'd want to make a new PR for this to finish it up and get it merged. Would that be ok?

**Comment by Dream-Master** [comment]
> @Dream-Master Considering OP seems to have gone missing, I'd want to make a new PR for this to finish it up and get it merged. Would that be ok?

Sure make a new pr

**Pull request closed without merging**